domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -1527,7 +1547,27 @@ QueryHelper buildSelectQueryHelper(
 
     QueryHelper queryHelper = new QueryHelper(transactionMode, table, facets);
 
-    queryHelper.selectQuery = SELECT + fieldName + addIdField + FROM + schemaName + DOT + table + SPACE + where;
+    String distinctOnClause = "";
+    if (distinctOn != null && !distinctOn.isEmpty()) {
+      String[] fields = distinctOn.split(",");
+      String distinctOnFields = Stream.of(fields)
+          .map(str -> String.format("lower(f_unaccent(%s))", str))
+          .collect(Collectors.joining(","));
+      distinctOnClause = String.format("DISTINCT ON (%s) ", distinctOnFields);
+    }
+
+    queryHelper.selectQuery = new StringBuilder()
```

Comment:
Use `+` instead of StringBuilder's `append` for increased readability. The compiler automatically uses a StringBuilder:
https://stackoverflow.com/questions/1532461/stringbuilder-vs-string-concatenation-in-tostring-in-java

Reply:
reverted to "+" style